climada/entity/exposures/crop_production.py

```diff
@@ -37,27 +37,32 @@
 logging.root.setLevel(logging.DEBUG)
 LOGGER = logging.getLogger(__name__)
 
-DEF_HAZ_TYPE = 'CP'
+DEF_HAZ_TYPE = 'RC'
```

Comment:
Why 'RC'?

Reply:
RC stands for "Relative Cropyield" (we changed the hazard from crop_potential to relative_cropyield).